apps/api/src/app/integrations/usecases/update-integration/update-integration.usecase.ts

```diff
@@ -123,11 +126,22 @@ export class UpdateIntegration {
       active: command.active,
     });
 
-    await this.invalidateCache.invalidateQuery({
-      key: buildIntegrationKey().invalidate({
-        _organizationId: command.organizationId,
-      }),
-    });
+    const isInvalidationDisabled = await this.getFeatureFlag.execute(
+      GetFeatureFlagCommand.create({
+        userId: command.userId,
+        environmentId: command.environmentId as string,
+        organizationId: command.organizationId,
+        key: FeatureFlagsKeysEnum.IS_INTEGRATION_INVALIDATION_DISABLED,
+      })
+    );
+
+    if (!isInvalidationDisabled) {
```

Comment:
Inversed this on purpose to make sure that false behaviour is the default for all customers and FF is not enabled in the environment